src/pages/index.js

```diff
@@ -90,4 +150,64 @@ const profileEditPopup = new PopupWithForm(
 );
 profileEditPopup.setEventListeners();
 
-const userInfo = new UserInfo(".profile__title", ".profile__description");
+function handleLikeIcon(card) {
+  if (card.isLiked) {
+    api
+      .removeLike(card.id)
+      .then(() => {
+        card.setIsLiked(false);
+      })
+      .catch((err) => console.error(err));
```

Comment:
Good job that you `catch` possible errors at the end of server requests. You can simplify it with `.catch(console.error);` because `console.error` is meant to log errors